ztp/ran-crd/site-config-crd.yaml

```diff
@@ -112,6 +112,9 @@ spec:
                         enum:
                           - OVNKubernetes
                           - OpenShiftSDN
+                      installConfigOverrides:
+                        description: InstallConfigOverrides allows to pass install config parameters to be added as annotation in the AgentClusterInstall CR.
```

Comment:
Maybe include the specific annotation in the description to help users understand where/what is affected.

Reply:
Will add in the description in the next fix.

Reply:
description added with commit 0843a14efa1eb6c8e7d476a1d7197a790758d76f